### ISSUE TITLE
Is `DeclarationRaws#value` property optional?

### ISSUE THREAD

**Issue by ybiquitous** [postcss/postcss#1730]
## Summary

The [`DeclarationRaws#value`](https://postcss.org/api/#declarationraws) property is required on the current PostCSS type definitions:

https://github.com/postcss/postcss/blob/0ab7a6ff80803ee4c9658a11d4932dcf5cc280bb/lib/declaration.d.ts#L21-L27

But actually, the property can be optional.

```js
const src = "a { background: linear-gradient(to right, #fff, #ddd) }";
postcss.parse(src).walkDecls((decl) => {
  console.log(decl.raws.value.raw);
                     // ↑ undefined
});
```

See the following reproduction for details.

## Reproduction

Type-checking passes, but running the code raises an error:

https://gist.github.com/ybiquitous/eabe11abd1ecb1feae79614ed12502bb

## Environment

PostCSS 8.4.8

**Comment by ai** [comment]
Do you want to send PR to save your name in project history?

**Comment by ybiquitous** [comment]
Yes, I’m interested. But it may take some time since I’m unfamiliar with the codebase.
On the other hand, I don’t want to block if someone can fix it quickly.

**Comment by ai** [comment]
Fixed 2e51425d.

**Comment by ai** [comment]
Released in 8.4.11.

**Comment by ybiquitous** [comment]
@ai Thank you for the quick fix! 😄
